### PR TITLE
Change to allow use of non-DF QURT drivers 

### PR DESCRIPTION
**Pull request by jywilson**
Mark,

Please merge these changes with integrate3 and then submit a pull request to PX4/main when you are ready.  

Jim W.

### CODE REVIEW

File: cmake/configs/qurt_eagle_release.cmake [hunk 1]
```diff
@@ -70,7 +70,7 @@ set(config_module_list
 	modules/uORB
 	modules/commander
 	modules/controllib
-
+
```
Comment (mcharleb) [inline cmake/configs/qurt_eagle_release.cmake]: Why?

File: src/modules/sensors/sensors.cpp [hunk 2]
```diff
@@ -2070,6 +2070,11 @@ Sensors::task_main()
 	/* start individual sensors */
 	int ret = 0;
 
+/*
```
Comment (mcharleb) [inline src/modules/sensors/sensors.cpp]: This change would preclude configs that did use DriverFramework drivers being used for a qurt build (there is more than one qurt build).

File: src/modules/sensors/sensors.cpp [hunk 3]
```diff
@@ -2215,15 +2221,21 @@ Sensors::task_main()
...
+				if (_gyro_sub[2] != -1) {
+					warnx("using tertiary gyro");
+					fds[0].fd = _gyro_sub[2];
+				}
 			} else {
-				fds[0].fd = _gyro_sub[1];
+				if (_gyro_sub[1] != -1) {
+					warnx("using secondary gyro");
+					fds[0].fd = _gyro_sub[1];
+				}
```
Comment (mcharleb) [inline src/modules/sensors/sensors.cpp:2256]: There should be some indication that main, tertiary and secondary failed or were not available.

File: src/platforms/qurt/include/qurt_log.h [hunk 4]
```diff
@@ -48,6 +48,8 @@ __EXPORT extern uint64_t hrt_absolute_time(void);
 // declaration to make the compiler happy.  This symbol is part of the adsp static image.
 void HAP_debug(const char *msg, int level, const char *filename, int line);
```
Comment (mcharleb) [inline src/platforms/qurt/include/qurt_log.h:49]: There should be a #pragma once added to the file, not the following check.

Done.

File: src/modules/sensors/sensors.cpp (hunk 3 above)
Comment (jywilson) [inline src/modules/sensors/sensors.cpp:2256]: Good point.  I will make the change.

File: cmake/configs/qurt_eagle_release.cmake (hunk 1 above)
Comment (jywilson) [inline cmake/configs/qurt_eagle_release.cmake]: Tried to remove this change, but couldn't.  Is there a git command to move a particular file to a previous commit?  I know the command to revert an entire commit.

File: src/modules/sensors/sensors.cpp (hunk 2 above)
Comment (jywilson) [inline src/modules/sensors/sensors.cpp]: Good point.  The DF stub class I created should solve the problem, but we need to get DF to successfully build with the QURT config.

File: cmake/configs/qurt_eagle_release.cmake (hunk 1 above)
Comment (mcharleb) [inline cmake/configs/qurt_eagle_release.cmake]:
```
git checkout <hash-of-commit-with-old-version> cmake/configs/qurt_eagle_release.cmake
```

File: src/modules/sensors/sensors.cpp (hunk 2 above)
Comment (jywilson) [inline src/modules/sensors/sensors.cpp]: Let me know if you will be working on the above problem.  If so, when you are done I will replace my changes to the driver that use the stub class.

**Pull request closed without merging**